students/20th/wonyoungcho/westagram/js/login.js

```diff
@@ -0,0 +1,20 @@
+'user strict';
+
+const id = document.querySelector('.loginName');
+const password = document.querySelector('.loginPassword');
+const loginBtn = document.querySelector('.loginButton');
+
+function login() {
+  if  ((id.value.includes('@')) && (3 < password.value.length)) {
```

Comment:
```suggestion
function login() {
  if  ((id.value.includes('@')) && (password.value.length > 3)) {
```
- 일반적으로 비교식을 사용할 때 변수가 왼쪽으로 오는게 자연스러워 보이네요